bazel_tools/ghc-lib/version.bzl

```diff
@@ -9,7 +9,7 @@ GHC_LIB_PATCHES = [
 ]
 
 GHC_REPO_URL = "https://github.com/digital-asset/ghc"
-GHC_REV = "81f87342aa67307891ba47e7425ac62d0de05917"
+GHC_REV = "011dfcfb53f125a4ad4629d9377e6e0514a04e15"
```

Comment:
Not super familiar with this process but shouldn't that be `a953391a059fc307a825aca42e23af8620c3ad7f` for now and then updated to whatever the merge commit ends up being?

Reply:
ah well spotted, thanks. `011dfcfb53f125a4ad4629d9377e6e0514a04e15` was the commit with `akrmn` as author; I've fixed it now